challenge_6.js

```diff
@@ -6,11 +6,32 @@
  */
 exports.SumOfASquare = SumOfASquare;
 
-function SumOfASquare(naturalNumbers){
+function SumOfASquare(naturalNumbers) {
+
   this.naturalNumbers = naturalNumbers;
+
+  this.sumOfSquares = function () {
+    let sumSquares = 0;
+    for (let i = 1; i < this.naturalNumbers + 1; i++) {
+      sumSquares += Math.pow(i, 2);
+    }
+    return sumSquares;
+  };
+
+  this.squareOfTheSums = function () {
+    let sum = 0;
+    for (let i = 1; i < this.naturalNumbers + 1; i++) {
+      sum += i;
+    }
+    return Math.pow(sum, 2);
+  }
 }
 
+
```

Comment:
From here down, you could probably just get rid of all the whitespace and comments.